src/Contexts/Adaptors/SubstrateApis/ChainBridgeAPI.tsx

```diff
@@ -64,12 +62,14 @@ export const getBridgeProposalVotes = async (
   decimalAmount: number
 ): Promise<GetBridgeProsalVotesRes | undefined> => {
   const destinationChainConfig = getСhainConfig(destinationChainId);
```

Comment:
@AndreiNavoichyk , can we remove the config from this lib `ChainBridgeAPI`? It shouldn't know anything about app config.